src/native/fs/fs_napi.cpp

```diff
@@ -2115,6 +2127,32 @@ set_debug_level(const Napi::CallbackInfo& info)
     return info.Env().Undefined();
 }
 
+/**
+ * register noobaa args to GPFS
+ */
+static Napi::Value
+register_gpfs_noobaa(const Napi::CallbackInfo& info)
+{
+    Napi::Object params = info[0].As<Napi::Object>();
+    struct gpfs_ganesha_noobaa_arg args = {
+        params.Get("version").ToNumber(),
+        params.Get("delay").ToNumber(),
+        params.Get("flags").ToNumber(),
+    };
+    LOG("FS::GPFS gpfs_ganesha_noobaa_arg=" << &args.noobaa_version);
+    LOG("FS::GPFS gpfs_ganesha_noobaa_arg=" << &args.noobaa_delay);
+    LOG("FS::GPFS gpfs_ganesha_noobaa_arg=" << &args.noobaa_flags);
```

Comment:
this seems to be printing pointers... no? also the message is not changing, maybe like this -
```suggestion
    LOG("FS::GPFS register noobaa for gpfs " 
        << DVAL(args.noobaa_version) 
        << DVAL(args.noobaa_delay) 
        << DVAL(args.noobaa_flags));
```